app/controllers/ems_infra_controller.rb

```diff
@@ -73,15 +73,14 @@ def scaling
     else
       scale_parameters_formatted = {}
       return_message = _("Scaling")
+      if @count_parameters.any? { |p| scale_parameters[p.name].to_s < p.value.to_s }
```

Comment:
Missing a check for `scale_parameters[p.name].nil?`, otherwise this will flash any time there's a `nil`.